.gitignore

```diff
@@ -27,4 +27,5 @@ nuget.exe
 project.lock.json
 /.vs/
 .build/
-.testPublish/
\ No newline at end of file
+.testPublish/
+launchSettings.json
```

Comment:
I am not doing this. .gitignores seems to vary by repo. Actually I checked a handful of projects and neither of them had launchSettings.json checked in and some of them already had launchSettings.json in .gitignore. I think .gitignore are typically copied when the repo is created and then they are updated as needed and keeping them consistent is not a goal.